editoast/src/schema/rolling_stock/mod.rs

```diff
@@ -34,8 +36,10 @@ crate::schemas! {
 
 pub const ROLLING_STOCK_RAILJSON_VERSION: &str = "3.2";
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ToSchema)]
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema, Derivative)]
+#[derivative(PartialEq)]
 pub struct RollingStockCommon {
+    #[derivative(PartialEq = "ignore")]
```

Comment:
why?

Reply:
it was also ignored in RollingStockModel, i guess it doesn't matter for comparing rolling stocks

Reply:
Most of the time, we don't use RollingStockModel directly, but rather the changeset, especially in tests. What I'm trying to do is to move the comparison logic from RollingStockModel, and I'm uncertain whether it can be moved to the Changeset.